portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +29,109 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function loadIntro(){
+    document.getElementById("play_button").style.visibility = "hidden"
+    //const info = "Hi I'm Cynthia. I'm a sophomore studying Computer Science and Statistics at Cornell University. I like photography and playing tennis and traveling. I was born and raised in New York and spend most of my time taking pictures, hanging out with my friends, or learning new things. I like to read and do my best to keep reading even when life gets hectic." ;
+    document.getElementById('introTitle').style.display = "block";
+    document.getElementById('introInfo').style.display = "block";
+    document.getElementById('profesh1').style.display = "block";
+    document.getElementById('profesh2').style.display = "block";
+    document.getElementById('profesh3').style.display = "block";
```

Comment:
Here you're using getElementById for each element in a list. If you needed to add another profesh link, you'd need to add another line here.

A more resilient way of doing this would be to make a class "profesh" which you've already done, and use a for loop to iterate over all elements returned by document.getElementsByClassName. That way if you add or remove a profesh link, you won't have to add another line to your javascript!